src/Ap.js

```diff
@@ -7,3 +7,4 @@
 //   ReactDOM.render(<App />, div);
 //   ReactDOM.unmountComponentAtNode(div);
 // });
+console.log('hello')
```

Comment:
I think you forgot a console here :ok_woman: 

Also, why did you renamed from `App.test.js` to `Ap.js` ?

Reply:
The App.test.js wasn't supposed to exist kk, it came with create-react-app, i forgot to delete it. 
And about the console, let's pretend it never existed kkk